src/components/molecules/NavigationIcons.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+	<div class="icons">
+		<Cart/>
+		<CartStatus/>
+		<Stick/>
+		<HamburgerButton/>
+		<Corss/>
```

Comment:
typo?

Reply:
If you are asking about cross then no, it has double s

Reply:
but it says 'Corss' in the code :D